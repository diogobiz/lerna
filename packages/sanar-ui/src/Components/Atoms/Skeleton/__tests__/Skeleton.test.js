
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESSkeleton from '../'

it('renders correctly', () => {
    const component = (
        <ESSkeleton>
            <h1>ESSkeleton</h1>
        </ESSkeleton>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
