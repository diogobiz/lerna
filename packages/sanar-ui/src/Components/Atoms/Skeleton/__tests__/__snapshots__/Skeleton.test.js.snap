// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-skeleton es-skeleton"
>
  <div
    className="ant-skeleton-content"
  >
    <h3
      className="ant-skeleton-title"
      style={
        Object {
          "width": "38%",
        }
      }
    />
    <ul
      className="ant-skeleton-paragraph"
    >
      <li
        style={
          Object {
            "width": undefined,
          }
        }
      />
      <li
        style={
          Object {
            "width": undefined,
          }
        }
      />
      <li
        style={
          Object {
            "width": "61%",
          }
        }
      />
    </ul>
  </div>
</div>
`;
